core/src/test/scala/org/broadinstitute/dsde/workbench/hamm/db/JobTableQueriesSpec.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.dsde.workbench.hamm.db
+
+import java.time.Instant
+
```

Comment:
Switched us back to using `scalatest` because scalike supports it and gives us a (sometimes) handy `AutoRollback` trait.